src/views/Article/Index.jsx

```diff
@@ -223,25 +278,31 @@ const Article = (props) => {
 };
 
 Article.propTypes = {
-  article: PropTypes.instanceOf(Object),
+  articleObj: PropTypes.instanceOf(Object),
```

Comment:
@youngestdj it's generally an antipattern to use the word `Object` et. al. in variable names

Reply:
Fixed